### PR TITLE
Remove drupal/jsonapi and update drupal 8.7.9

### PR DESCRIPTION
**Pull request by anthony-malkoun**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-3272

### Changed

1.  Update Drupal to 8.7 and other updates as required.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,8 +4,7 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_core": "^1.2.5",
-        "drupal/jsonapi": "^2.4",
```
Comment (anthony-malkoun) [inline composer.json]: `drupal/jsonapi` is now part of Drupal core from 8.7.

File: composer.json [hunk 2]
```diff
@@ -4,8 +4,7 @@
     "type": "drupal-module",
     "license": "GPL-2.0-or-later",
     "require": {
-        "dpc-sdp/tide_core": "^1.2.5",
```
Comment (anthony-malkoun) [inline composer.json]: This will change when there is a release of `tide_core`

**Comment by vincent-gao** [comment]
1. use https://github.com/dpc-sdp/dev-tools/blob/feature/SDPA-3272-update-drupal-8-7/composer.dev.json#L9
2. delete jsonapi dependency from `tide_api.info.yml`

File: tide_api.info.yml [hunk 3]
```diff
@@ -4,7 +4,6 @@ type: module
 package: Tide
 core: 8.x
 dependencies:
-  - drupal:jsonapi (>=8.x-2.4)
```
Comment (anthony-malkoun) [inline tide_api.info.yml]: 👍

**Pull request merged**